Game/Pregnancy/EggCell.gd

```diff
@@ -168,3 +187,5 @@ func loadData(data):
 	motherSpecies = SAVE.loadVar(data, "motherSpecies", ["feline"])
 	resultSpecies = SAVE.loadVar(data, "resultSpecies", [])
 	resultGender = SAVE.loadVar(data, "resultGender", NpcGender.Male)
+	monozygotic = SAVE.loadVar(data, "monozygotic", 1)
+	_fetusReadyForBirth = SAVE.loadVar(data, "fetusReadyForBirth", false)
```

Comment:
Please don't begin the variables with _ in the future. Looks bad when some use it and some don't like here